portfolio/src/main/webapp/script.js

```diff
@@ -15,17 +15,34 @@
 /**
  * Adds a random fun fact about myself to the page.
  */
-function addRandomFact() {
+function addFunFact() {
   const facts =
       ['I have two older brothers!',
        'I\'ve played tennis for fourteen years!', 
        'My favorite book is Enlightenment Now by Steven Pinker.', 
        'My favorite TV show is It\'s Always Sunny in Philadelphia.'];
 
-  // Pick a random fact.
+  // Pick a random fun fact.
   const fact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
+
+  // Add 'Hide fun fact' button to the page.
+  const hideFactButton = document.getElementById('fun-fact-hide');
+  hideFactButton.style.display = 'inline-block';
+}
+
+/**
+ * Hides the fun fact from user.
+ */
+function hideFunFact() {
+    // Remove it from the page. 
+    const factContainer = document.getElementById('fact-container');
+    factContainer.innerText = '';
+
+    // Remove 'Hide fun fact' button from the page.
+    const hideFactButton = document.getElementById('fun-fact-hide');
+    hideFactButton.style.display = 'none';
```

Comment:
Something to consider is that this block is pretty similar to the above section (lines 28 thru 34), since both look up and update the same DOM elements. At this point I think that's fine but if you're planning to add more functionality to this part of the page it may be worth considering how to restructure this (eg: adding helper methods) so that you don't need to repeat the element ids and assignments in two places.

To be clear, no action required on this comment.